packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -444,6 +460,21 @@ function TableVirtualizer({layout, collection, focusedKey, renderView, renderWra
         onScroll={onScroll}>
         {state.visibleViews[1]}
       </ScrollView>
+      <div
+        aria-hidden
+        className={classNames(styles, 'spectrum-Table-colResizeIndicator')}
+        style={{
+          [direction === 'ltr' ? 'left' : 'right']: `${resizerPosition}px`,
+          visibility: columnState.currentlyResizingColumn != null ? 'visible' : 'hidden'
+        }}>
+        <div
+          className={classNames(styles, 'spectrum-Table-colResizeNubbin')}
+          style={{
+            visibility: resizeModeState[0] ? 'visible' : 'hidden'
```

Comment:
You may have already talked w/ design regarding this but I was wondering if might be nice to have the nubbin also appear in mouse drag for consistency?

Reply:
Ok, so this was a conversation with @devongovett, I'm happy to change it. But his interpretation was that the Nubbin only appears if the menu was used to trigger the resize.

Reply:
A clarifying question would be what the exact purpose of the Nubbin is. If it is going to change its icon to an arrow that points to the left/right when you hit the max/min then it might be nice to have in the mouse drag case as well since the mouse cursor only shows when you are hovering above/around the resizer element. The mouse cursor actually gets overwritten if you move your mouse below the resizer during the drag or if you hover an element outside the table. If the Nubbin won't dynamically change its icon then I'm fine with the experience as is

Reply:
Looks good to me now, the mouse cursor behaves much nicer now